File: app/src/main/java/com/example/advancedprayertimes/Logic/Entities/API_Entities/Diyanet/DiyanetSehirEntity.java
package com.example.advancedprayertimes.Logic.Entities.API_Entities.Diyanet;

import com.google.gson.annotations.SerializedName;

public class DiyanetSehirEntity
{
    // region fields

    @SerializedName("SehirID")
    private String _sehirID = null;

    @SerializedName("SehirAdiEn")
    private String _sehirAdiEn = null;

    // endregion fields

    // region getter & setter

    public String getSehirID()
    {
        return _sehirID;
    }

    public void setSehirID(String sehirID)
    {
        _sehirID = sehirID;
    }

    public String getSehirAdiEn()
    {
        return _sehirAdiEn;
    }

    public void setSehirAdiEn(String sehirAdiEn)
    {
        _sehirAdiEn = sehirAdiEn;
    }

    // endregion getter & setter
}

File: app/src/main/kotlin/com/example/advancedprayertimes/logic/api_entities/diyanet/DiyanetSehirEntity.kt
package com.example.advancedprayertimes.logic.api_entities.diyanet

import com.google.gson.annotations.SerializedName

class DiyanetSehirEntity : AbstractDiyanetSubEntity() {

    @SerializedName("SehirID")
    var sehirID: String? = null

    @SerializedName("SehirAdiEn")
    var sehirNameEn: String? = null

    override val id: String?
        get() = sehirID

    override val nameEn: String?
        get() = sehirNameEn
}